### ISSUE TITLE
After recent upgrade to version 2.0.0, AppTester is not working anymore

### ISSUE THREAD

**Issue by raffaeu** [zapier/zapier-platform-cli#101]
I have an App using Zapier CLI 1.0.x and today I decided to upgrade to the latest version, and suddenly all integration tests working with `appTester` are not working anymore.

> After the App is deployed, everything is still working on Zapier.com, the problem occurs only with AppTester while running the mocha tests.

These are the steps I executed for the upgrade:

- [x] Upgraded Node to 6.10.2
- [x] Upgraded zapier-platform-core to 2.0.0
- [x] Upgraded Mocha to 3.4.2

Now, if I try to execute any test with appTester, for example like this one which is working fine on Zapier 1.0.1, I get an odd error from the Zapier Core Framework:

```javascript
    it.('should authenticate service account', (done) => {

        const bundle = {
            authData: {
                username: process.env.DEMO_USERNAME,
                password: process.env.DEMO_PASSWORD,
                tenant_id: process.env.DEMO_TENANT
            }
        };

        appTester(Auth.authentication.sessionConfig.perform, bundle)
            .then((result) => {
                result.access_token.should.not.be.null();
                result.refresh_token.should.not.be.null();
                result.username.should.not.be.null();
                result.user_id.should.not.be.null();
                result.appUserId.should.not.be.null();
                done();
            })
            .catch(done);
    });
```

I get a very strange exception which I don't understand where is coming from:

```bash
Unhandled rejection SyntaxError: Invalid regular expression: /C:\Program Files\Microsoft MPI\Bin\/: \ at end of pattern
    at RegExp (native)
    at Object.keys.forEach (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\cleaner.js:38:18)
    at Array.forEach (native)
    at replacer (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\cleaner.js:36:23)
    at recurseReplace (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\data.js:99:11)
    at Object.recurseReplaceBank (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\cleaner.js:43:10)
    at sendLog (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\create-logger.js:89:30)
    at logErrorAndCallbackOnce (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\create-lambda-handler.js:60:7)
    at Object.app.then.catch (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\zapier-platform-core\src\tools\create-lambda-handler.js:93:11)
    at bound (domain.js:280:14)
    at Object.runBound (domain.js:293:12)
    at Object.tryCatcher (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\promise.js:689:18)
    at Async._drainQueue (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (C:\DEV\GITHUB\ServiceHubCore\src\Mp.Sh.Zapier\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:666:20)
    at tryOnImmediate (timers.js:639:5)
    at processImmediate [as _immediateCallback] (timers.js:611:5)
```

**Comment by raffaeu** [comment]
@xavdid introduced the bug by modifying the regular expression on `cleaner.js`
https://github.com/zapier/zapier-platform-core/commit/8dd8c36281c388f0116e310e689ffe5e6b913976#diff-08dbe273769bf2e845e4067724ac64dd

**Comment by xavdid** [comment]
Ok @raffaeu I believe I have a fix here: https://github.com/zapier/zapier-platform-core/tree/windows-fix

Can you `npm install` using that branch for core and confirm everything works? I tested on my windows machine and my tests passed, but I was using a sample app. Once you give me the 👍, I can merge and it'll go out with the next deploy.

**Comment by raffaeu** [comment]
@xavdid awesome job, fixed in less than a day. I love this new **Agile** approach.
Yes, I run all my tests on _local_ and on _team city_ and everything is working fine.

I used this:

```bash
$ npm install --save git://github.com/zapier/zapier-platform-core.git#windows-fix
mp-sh-zapier@0.1.0 C:\xyz
`-- zapier-platform-core@2.0.0  (git://github.com/zapier/zapier-platform-core.git#24b32be7bd63114d80111dc673733ecccbf59dff)
```

**Comment by xavdid** [comment]
Sweet, glad to hear it! I'll get that merged and we'll do a deploy in the next couple of days (after July 4th) and then you'll be able to use the eventual `2.0.1` from npm as normal. 

Thanks for the report here - none of us have windows machines, so it's tough to catch this sort of thing in development.

**Comment by raffaeu** [comment]
@xavdid not a problem at all, for now I can just check-in and deploy so I can continue like this for days 😄 

> I like this way of working, for me it's way better than the visual tool because I have my test harnesses, I can work with `node.js` and create a good ETL integration with your platform.

**Comment by xavdid** [comment]
Sweet. The fix is in `core`'s master. I'd switch off the feature branch when you have a chance so if we delete it, we don't mess up your build. 

And I'm glad you like it! We agree that it's a huge improvement over the web builder for technical folks.

**Comment by raffaeu** [comment]
Ok I can see the merge, please turn off the branch I'll switch right away.
Thanks

**Comment by xavdid** [comment]
@raffaeu FYI, this is released as `2.0.1` and you can move whenever is convenient.

**Comment by raffaeu** [comment]
I found it and I have already upgraded my app.

Thank you
